Lesson Plans/Flight Instructor/CFI/CFI1.2 Effective Communication.md

```diff
@@ -0,0 +1,109 @@
+### Objective
+Understand the characteristic of adult learners, aware of the basic elements of the communications and how to develop communication skills towards flight instruction.
+
+### Resources
+- [[CFI ACS]] Section FI.I.A
+- [[Aviation Instructors Handbook]] Chapter 2, 4
+
+### Builds On
+None
+
+### Schedule
+Ground 0.5 hours
+
+### Lesson Elements
+1. Teaching the adult learner
+	1. Adult Learner Characteristics
+		1. Motivated to learn primarily because they have a use for the knowledge or skills sought. *Learning is a means to an end, not an end in itself.*
+		2. Seek out learning experience due to life-changing events (marriage, job, etc.)
+		3. Autonomous and self-directed; *They want independent and self-control.*
+		4. Accumulated life experience and knowledge to draw from
+		5. Goal oriented
+		6. Relevancy Oriented
+		7. Practical, focusing on what is useful for them
+		8. *As do all learners,* want to be shown respect
+		9. Need to increase or maintain self-esteem is a strong secondary motivator
+		10. Want to solve problems and apply new knowledge immediately
+	2. What instructors should do
+		1. Provide an organized training syllabus with clearly stated objectives
+		2. Help learners integrates new ideas *with what they already know*
+		3. Clarify and articulate all learner expectations early on
+		4. Recognize learner's need to control pace and start/stop time
+		5. Use frequent scenario-based training
```

Comment:
I feel like Scenario-Based Training is important enough for it's own concept page. The FAA _loves_ scenario based, and honestly I do too now..